google/cloud/storage/client.cc

```diff
@@ -481,6 +483,19 @@ std::string CreateRandomPrefixName(std::string const& prefix) {
                                                   "abcdefghijklmnopqrstuvwxyz");
 }
 
+std::string Client::Endpoint() const {
+  return connection_->options().get<RestEndpointOption>();
+}
+
+std::string Client::Host() const {
```

Comment:
No action required: my instinct is to try to optimize this (too many string copies), but then I remember that this code is used very rarely.  Maybe a comment reminding  the performance obsessed (like myself) that this code is not in any critical path.

Reply:
Ack.